stories/Atom/Form/Radio/radio.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/breakpoints";
+
+.radio-wrapper {
+    position: relative;
+    padding-left: 20px;
+    display: inline-block;
+    input[type=radio] {
+        top: 2px;
+        left: 0;
+        opacity: 0;
+        position: absolute;
+        & + label {
+            cursor: pointer;
+            &:hover::before {
+                border-color: $color-light-pink;
+            }
+            &::before {
+                left: 0;
+                top: 2px;
+                content: "";
+                width: 12px;
+                height: 12px;
+                position: absolute;
+                border-radius: 50%;
+                border: 2px solid $color-dark-red;
+            }
+        }
+        &:checked + label::after {
+            top: -3px;
+            left: 3px;
+            content: "•";
+            color: $color-dark-red;
+            font-size: 19px;
+            position: absolute;
+        }
+    }
+}
```

Comment:
@Shubham-Sharma-77 We can create the placeholders for some part of styling is same for radio and checkbox